packages/turf-point-grid/types.ts

```diff
@@ -1,10 +1,11 @@
-import { BBox } from '@turf/helpers';
+import { BBox, polygon, Polygon} from '@turf/helpers';
 import pointGrid from './';
 
 const cellSide = 50;
 const bbox: BBox = [-95, 30, -85, 40];
+const poly: Polygon = polygon([[[20, 30], [10, 10], [20, 20], [20, 30]]]);
```

Comment:
@stebogit This is my bad... `Polygon` was short for `Feature<Polygon>`, however now `Polygon` from `@turf/helpers` is now equivalent to `Geometry<Polygon>`.

The reason for the error is because `mask` doesn't handle adding `Geometry<Polygon>` (which should be a valid input, both Geometry & Feature).